### ISSUE TITLE
The "backup.ark.heptio.com/backup-volumes" setting for restic is unclear

### ISSUE THREAD

**Issue by trondhindenes** [heptio/ark#568]
Not sure if this should point to the name of a pvc, the name of a volume, or the name of the mounted path inside the pod. Would be awesome to get that clarified

**Comment by trondhindenes** [comment]
It would be great to get better logging of what ark expected to find, and what it didn't find. If there's an expected metadata annotation that is missing maybe log that out aswell. Reading the statement below its still not clear to me where the error is.
```
time="2018-06-21T22:02:52Z" level=error msg="backup failed" error="pod volume backup failed: error getting volume directory name: volume not found in pod" key=heptio-ark/nginx-example logSource="pkg/controller/backup_controller.go:280"
```

**Comment by skriss** [comment]
Good feedback. FYI this should be a comma-separated list of volume names.

**Comment by rosskukulinski** [comment]
@skriss Marking this against the v0.9.0 milestone, let me know if that doesn't feel right to you.

Two action items I think:

* Improve log output to aid debuggability
* Improve [documentation](https://github.com/heptio/ark/blob/master/docs/restic.md#run) for the field and/or provide a Deployment manifest which has the inline annotation.

**Comment by skriss** [comment]
The documentation has been clarified. I'll tweak the log message before v0.9.0.

**Comment by skriss** [comment]
Actually improved logging has already been fixed as well, with #606. Closing this issue out, but please reach out if any further issues!
